### PR TITLE
LB-152: Use new schema when deduplicating listens in influx-writer

### PR DESCRIPTION
**Pull request by paramsingh**
We missed updating the influx-writer code when we changed the influx schema.

### CODE REVIEW

**Comment by paramsingh** [comment]
How would I go about writing tests for dedup? There are no values returned anywhere.

File: influx-writer/influx-writer.py [hunk 1]
```diff
@@ -36,6 +36,12 @@ def __init__(self, ls, influx, redis):
         self.inserts = 0
         self.time = 0
 
+    @staticmethod
```
Comment (mayhem) [inline influx-writer/influx-writer.py]: We already have this somewhere -- lets not duplicate this and put this into a common module please.

File: influx-writer/influx-writer.py [hunk 2]
```diff
@@ -58,17 +64,13 @@ def write(self, listen_dicts):
...
         # quering for artist name here, since a field must be included in the query.
         query = """SELECT time, artist_name
-                     FROM listen
-                    WHERE user_name = '%s'
-                      AND time >= %d000000000
+                     FROM "\\"%s\\""
+                    WHERE time >= %d000000000
                       AND time <= %d000000000
                 """ % (user_name, min_time, max_time)
+
```
Comment (mayhem) [inline influx-writer/influx-writer.py]: Hmm. Does the user name need escaping here?

Comment (paramsingh) [inline influx-writer/influx-writer.py]: I did it how it was done here: https://github.com/metabrainz/listenbrainz-server/blob/master/listenstore/influx_listenstore.py#L199

I guess we should escape the username there also, I'll fix and send soon.

File: influx-writer/influx-writer.py (hunk 1 above)
Comment (paramsingh) [inline influx-writer/influx-writer.py]: Would a `utils.py` file in the root folder be acceptable?

Comment (mayhem) [inline influx-writer/influx-writer.py]: For now, that is fine. Ideally I would like to get rid of all python modules (not scripts) from root folder... But we can tackle that later.

**Comment by paramsingh** [comment]
Sorry, I haven't been able to get to fixing this, been swamped with lots of stuff, I'll try to do it this week.

**Comment by paramsingh** [comment]
Test added, moved the escape code to `listenstore/utils.py` and  escaped everything in `influx-writer.py` and `InfluxListenStore`. I think everything's fixed now, please take a look again :)

File: tests/integration/test_influx_writer.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        r = self.send_single_listen(user)
+        self.assert200(r)
+        time.sleep(5)
+        r = self.send_single_listen(user)
+        self.assert200(r)
+        time.sleep(5)
+
+        to_ts = int(time.time())
+        listens = self.ls.fetch_listens(user['musicbrainz_id'], to_ts=to_ts)
+        print(listens)
```
Comment (mayhem) [inline tests/integration/test_influx_writer.py]: i guess this isn't really needed here. I'll take care of removing it.

**Pull request merged**